sdk/communication/Azure.Communication.CallingServer/src/Models/CallMediaRecognizeOptions.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Communication.CallingServer
+{
+    /// <summary>
+    /// Optional elements for Recognize.
+    /// </summary>
+    public class CallMediaRecognizeOptions
+    {
+        /// <summary>
+        /// Initializes a RecognizeOptions object.
+        /// </summary>
+        /// <param name="recognizeInputType"></param>
+        /// <param name="recognizeConfigurations"></param>
+        public CallMediaRecognizeOptions(RecognizeInputType recognizeInputType, RecognizeOptions recognizeConfigurations)
+        {
+            RecognizeInputType = recognizeInputType;
+            RecognizeOptions = recognizeConfigurations;
+        }
+        /// <summary>
+        /// Recognize Input Type.
+        /// </summary>
+        public RecognizeInputType RecognizeInputType { get; }
+
+        /// <summary>
+        /// Recognize Configurations.
+        /// </summary>
+        public RecognizeOptions RecognizeOptions { get; }
+
+        /// <summary>
+        /// Should stop current Operations?.
```

Comment:
Are bools usually worded as positive or negative statements? This reads more as a statement than a bool. Suggestion would be "AllowRecognitionInterruption" or something similar.

Reply:
I'll sync internally about naming.